### PR TITLE
Feat/action buttons

### PR DESCRIPTION
**Pull request by ferreroltd**
This PR adds the first action button and exposes the 2 new composed components that are 
TextareaCopy:
image
image

and InputCopy

image
image

### CODE REVIEW

File: packages/libs/react-ui/src/components/Form/Input/Input.tsx [hunk 1]
```diff
@@ -78,6 +79,7 @@ export const Input: FC<IInputProps> = forwardRef<HTMLInputElement, IInputProps>(
             disabled={disabled}
             {...rest}
           />
+          <div>{children}</div>
```
Comment (ferreroltd) [inline packages/libs/react-ui/src/components/Form/Input/Input.tsx:82]: the default position looked okay

File: packages/libs/react-ui/src/components/Form/Input/InputCopy/InputCopy.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import type { IInputProps } from '@components/Form';
+import { InputCopy } from '@components/Form';
+
+import type { SystemIcon } from '@components/Icon';
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+const meta: Meta<IInputProps> = {
+  title: 'Form/InputCopy',
```
Comment (eileenmguo) [inline packages/libs/react-ui/src/components/Form/Input/InputCopy/InputCopy.stories.tsx:9]: Could we add another folder to group all input stories together? I think you can just update this to 'Form/Input/InputCopy'

**Review by eileenmguo — COMMENTED** [review]
Some feedback discussed during our call 
image

Also the button looks different from the design. I would recommend creating a custom button

File: packages/libs/react-ui/src/components/Form/Textarea/Textarea.tsx [hunk 3]
```diff
@@ -46,6 +47,7 @@ export const Textarea: FC<ITextareaProps> = forwardRef<
           disabled={disabled}
           {...rest}
         />
+        <div className={buttonContainerClass}>{children}</div>
       </div>
```
Comment (eileenmguo) [inline packages/libs/react-ui/src/components/Form/Textarea/Textarea.tsx:51]: Can we combine the two divs?

**Comment by realdreamer** [comment]
image
image

The icons seems bit off with the size. It's smaller than the current implementation

**Review by sstraatemans — APPROVED** [review]

**Review by salamaashoush — APPROVED** [review]

**Review by eileenmguo — APPROVED** [review]

**Pull request merged**